docs/trace-agent/datadog.example.yaml

```diff
@@ -19,6 +19,13 @@ api_key: 1234
 # 
 # # hostname
 # hostname: localhost
+#
+# # The host to listen on for metrics and traces. This is ignored when non-local
+# # traffic is enabled on either APM or dogstatsd. The trace-agent uses this host
```

Comment:
it'd be nice to clarify that enabling non-local traffic on APM doesn't make Dogstatsd ignore this setting (and the other way around as well), but not sure how to better communicate this 🤔

Reply:
I went full explicit here to be clear. WDYT?